### PR TITLE
Docs: add section for lava-docker

### PR DESCRIPTION
**Pull request by aliceinwire**
Signed-off-by: Alice Ferrazzi <alice.ferrazzi@miraclelinux.com>

### CODE REVIEW

**Comment by aliceinwire** [comment]
Fixed commit title

**Review by gctucker — COMMENTED** [review]
`lava-docker` is a GitHub repository, so it's more a piece of software than a service.  As such I believe it should be moved under the "software maintainers" section rather than services.

**Comment by aliceinwire** [comment]
> `lava-docker` is a GitHub repository, so it's more a piece of software than a service. As such I believe it should be moved under the "software maintainers" section rather than services.

@gctucker fixed

**Review by gctucker — APPROVED** [review]

**Comment by gctucker** [comment]
Tested OK on staging https://static.staging.kernelci.org/docs/team/tsc/#lava-docker

File: kernelci.org/content/en/docs/team/tsc.md [hunk 1]
```diff
@@ -92,6 +92,12 @@ KCIDB provices a set of tools to submit kernel test data to a common database.
 * Grafana dashboard: [`kcidb-grafana`](https://github.com/kernelci/kcidb-grafana)
 * Maintainer: `spbnick`
 
+### lava-docker
+
+Simplify the install and maintenance of a LAVA lab using docker container.
+
+* Repository: [`kernelci-core`](https://github.com/kernelci/lava-docker)
```
Comment (gctucker) [inline kernelci.org/content/en/docs/team/tsc.md:99]:
```suggestion
* Repository: [`lava-docker`](https://github.com/kernelci/lava-docker)
```

File: kernelci.org/content/en/docs/team/tsc.md [hunk 2]
```diff
@@ -92,6 +92,12 @@ KCIDB provices a set of tools to submit kernel test data to a common database.
...
+* Maintainer: `alicef`, `montjoie`
```
Comment (gctucker) [inline kernelci.org/content/en/docs/team/tsc.md:100]:
```suggestion
* Maintainers: `alicef`, `montjoie`
```

**Review by gctucker — CHANGES_REQUESTED** [review]
Just saw a couple of minor things to tweak

**Comment by aliceinwire** [comment]
> Just saw a couple of minor things to tweak

fixed thanks

**Review by gctucker — APPROVED** [review]
Thanks, checked OK again on staging.

**Pull request merged**